include/conversation.php

```diff
@@ -394,10 +394,9 @@ function visible_activity($item) {
 		}
 	}
 
-	if (activity_match($item['verb'], ACTIVITY_FOLLOW) && $item['object-type'] === ACTIVITY_OBJ_NOTE) {
-		if (!($item['self'] && ($item['uid'] == local_user()))) {
-			return false;
-		}
+	/// @TODO such things are unneccessary code-complexibilities and should be avoided
+	if (activity_match($item['verb'], ACTIVITY_FOLLOW) && $item['object-type'] === ACTIVITY_OBJ_NOTE && !(x($item, 'self') && ($item['uid'] == local_user()))) {
```

Comment:
Please use `empty()` instead of `!x()`.

Reply:
So ` && empty($item['self']) && $item['uid'] == local_user()` is correct here?

Reply:
Yes.